MdePkg/MdePkg.dec

```diff
@@ -362,6 +362,10 @@
   #
   ArmMmuLib|Include/Library/ArmMmuLib.h
 
+  ##  @libraryclass  Provides a default exception handler.
+  #
+  DefaultExceptionHandlerLib|Include/Library/DefaultExceptionHandlerLib.h

```

Comment:
This is gated under `[LibraryClasses.ARM, LibraryClasses.AARCH64]`, but still being in MdePkg.dec I would consider renaming this to be more specific or merge it with other existing libraries classes, if applicable

Reply:
3 commits has been created to address this commit:
1. [CpuExceptionHandlerLib: Move DumpImageAndCpuContent to library header file]
2. [ArmPkg: Merge DefaultExceptionHandlerLib with ArmExceptionLib]
3. [ArmPkg,UefiCpuPkg: Move ArmExceptionLib from ArmPkg to UefiCpuPkg]